README.md

```diff
@@ -59,10 +59,22 @@ cmake --build .
 ```
 
 ## Usage
-Before proceeding, the Linux driver must be unloaded. On the Talos II, this can be done as root with the following:
+Before proceeding, the Linux driver must be unloaded. 
+
+If the driver is loaded as a module, unload it (as root):
 ```bash
-echo 0004:01:00.0 > /sys/bus/pci/devices/0004:01:00.0/driver/unbind
-echo 0004:01:00.1 > /sys/bus/pci/devices/0004:01:00.1/driver/unbind
+modprobe -r tg3
```

Comment:
I see, I didn't understand what unbinding was needed for. In that case, since you said it's possible to update it without unbinding, would it be preferable to use that method? If I understand correctly, since the APE firmware can be reflashed with the old FW and the rewrite is stable, there's not much reason to test it in RAM except for development, and that procedure can be relegated to a later section.

If not, I would suggest just unbinding one function in the usage procedure, though if the rationale is "maintain an SSH connection" you'd need to make sure the user knows to pick a port they're not using, which complicates the setup (now you introduce a variable that you have to fill in for the flashing steps, etc.)...

Maybe this is all moot if fwupd ends up being the preferred way to flash, though.

Reply:
Good point. I'll definitely update this one fwupd is official released and in some channels to how the new method.

You're also correct that the if the released code is used, then there should be no issue with just flashing it on the Talos II and Blackbird, so the comments about testing it first can be removed.

Let me know if you want to make changes to the PR, otherwise I can go ahead and update the README as appropriate.

Reply:
Cool! I'll go ahead and make another PR, I think. Would you like me to make the "usage" section look more like the release zip (where all the utils are in bin/), and move the "compiling" section below it, with the test command syntax moved there? If not, that's fine, I just wanna make sure it's appropriately tailored for the audience you think it's intended for.

Reply:
Also, am I correct in assuming that flashing using `-i eth` will not interrupt network traffic on that port?

Reply:
Correct, network traffic will not be interrupted.